site/src/_layouts/default.hbs

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+
+    <title>d3fc {{#if title}}:{{/if}} {{title}}</title>
```

Comment:
I've compromised at 
```
<title>d3fc{{#if title}}:{{title}}{{/if}}</title>
```

The `d3fc` bit is short enough that I don't think we need to reverse the order.